lib/Projects/ProjectGitSyncer.php

```diff
@@ -70,6 +70,17 @@ public function checkoutBranch(string $repositoryName, string $branchName): void
         $this->processFactory->run($command);
     }
 
+    public function checkoutTag(string $repositoryName, string $tagName): void
+    {
+        $command = sprintf(
+            'cd %s && git clean -xdf && git checkout tags/%s',
+            $this->getRepositoryPath($repositoryName),
+            $tagName
```

Comment:
Although it will be used only for doctrine-owned projects, it still makes sense to implement proper escaping of the command line arguments.

Reply:
Good point. Done.